enforcers/vm_enforcer/shell/install_vme.sh

```diff
@@ -78,11 +78,14 @@ is_it_rhel() {
   cat /etc/*release | grep PLATFORM_ID | grep "platform:el8" &>/dev/null
 
   if [ $? -eq 0 ]; then
-    echo "Info: This is RHEL 8 system. Going to apply SELinux policy module"
-
+    echo "Info: This is RHEL 8 system. Going to download and apply SELinux policy module"
+    echo "Info: Downloading SELinux policy module"
+    wget -q https://raw.githubusercontent.com/aquasecurity/deployments/2022.4/enforcers/vm_enforcer/rpm/selinux/aquavme/aquavme.te
```

Comment:
Can this be replaced with curl -o ? Otherwise the customer will need to install wget

Reply:
I've changed it to Curl and added RHEL8's SElinux prerequisites to the README file